src/popup/components/Layout/Fullscreen/Onboarding.tsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import styled from "styled-components";
```

Comment:
unfortunately, the recover account design breaks the design system all the other Onboarding pages follow. had to do some refactoring here, but i fixed all the pages that use this stuff